lib/CL/clCreateProgramWithIL.c

```diff
@@ -37,7 +37,8 @@
 static int
 get_program_spec_constants (cl_program program, char *program_bc_spirv)
 {
-  char *args[] = { LLVM_SPIRV, "--spec-const-info", program_bc_spirv, NULL };
+  char *args[] = {get_llvm_spirv(), "--spec-const-info", program_bc_spirv,
```

Comment:
Did you run clang-format (format-diff/branch.sh)? Not sure if it prefers to have the white space in the beginning/end of the list.

Reply:
I ran `git clang-format`, assuming that there would be a top-level `.clang-format`. Apparently the workflow here is different, I'll re-format the changes.

Reply:
Yep, unfortunately we have different styles for C and C++.